management/server/http/api/openapi.yml

```diff
@@ -450,44 +450,85 @@ components:
               enum: [ "name","description","disabled","flow","sources","destinations" ]
           required:
             - path
-    PolicyRule:
+    PolicyRuleMinimum:
       type: object
       properties:
         id:
-          description: Rule ID
+          description: Policy rule ID
           type: string
         name:
-          description: Rule name identifier
+          description: Policy rule name identifier
           type: string
         description:
-          description: Rule friendly description
+          description: Policy rule friendly description
           type: string
         enabled:
-          description: Rules status
+          description: Policy rule status
           type: boolean
-        sources:
-          description: policy source groups
-          type: array
-          items:
-            $ref: '#/components/schemas/GroupMinimum'
-        destinations:
-          description: policy destination groups
-          type: array
-          items:
-            $ref: '#/components/schemas/GroupMinimum'
         action:
-          description: policy accept or drops packets
+          description: Policy rule accept or drops packets
           type: string
           enum: ["accept","drop"]
+        bidirect:
```

Comment:
it will be better to use bidirectional, also the description:
```yaml
        bidirectional:
          description: Define if the rule is applicable in both directions, sources, and destinations.
          type: boolean
```

Reply:
Fixed